plugins/ggdm30_rules.js

```diff
@@ -6507,6 +6508,7 @@ ggdm30.rules = {
     'DPAC':['DPAL','DPAU'],
     'GSGCHC':['GSGCHL','GSGCHU'],
     'GSGCLC':['GSGCLL','GSGCLU'],
+    'PFHC':['PFHL','PFHU'],
```

Comment:
what's this special mapping do?

Reply:
That was something I missed in the translation.  In TDS6 & GGDM, there are these type of just plain ugly attributes:
```
PFHC	Predominant Feature Height <interval closure>
  2 = Open Interval	The bounded open interval (minimumValue, maximumValue).
  3 = Greater-than-or-equal to Less-than Interval	The right half-open bounded interval [minimimValue, maximumValue).
  4 = Greater-than to Less-than-or-equal Interval	The left half-open bounded interval (minimimValue, maximumValue].
  5 = Closed Interval	The bounded interval [minimimValue, maximumValue].
  6 = Greater-than Semi-interval	The left half-open unbounded interval (minimumValue, +infinity).   [Description] The value supplied for the upper endpoint, if any, is not meaningful.
  7 = Greater-than or Equal Semi-interval	The unbounded interval [minimumValue, +infinity).   [Description] The value supplied for the upper endpoint, if any, is not meaningful.
  8 = Less-than Semi-interval	The right half-open unbounded interval (-infinity, maximumValue).   [Description] The value supplied for the lower endpoint, if any, is not meaningful.
  9 = Less-than or Equal Semi-interval	The unbounded interval (-infinity, maximumValue].   [Description] The value supplied for the lower endpoint, if any, is not meaningful.

PFHL	Predominant Feature Height <lower value>
PFHU	Predominant Feature Height <upper value>
```

The default value for the `interval closure` attributes is `5`. So, on export it is set to `5` and on import it would create an OSM tag for it - `height:feature:closure=closed_interval`

There is a function that reads that table and says: If the Upper and Lower values are default's, 
 then delete the closure tag.  I missed adding `PFHC` to the list.